### PR TITLE
No longer timestamping files (Fixes #13)

### LINKED ISSUE #13: Do not timestamp files
Having a timestamp could be bad if they want to correlate your images to being somewhere at a certain time. (A hash of the encrypted file will suffice for a unique ID but an increasing number is also OK.)

Bonus: modify the last modified time of the file.

### PR DESCRIPTION
**Pull request by DrDub**
Replaced the timestamping with sequential integers.

### CODE REVIEW

File: app/src/main/java/ca/deflector/passable_shots/MainActivity.java [hunk 1]
```diff
@@ -44,7 +45,20 @@ public void onPictureTaken(byte[] data, Camera camera) {
                 // call encryption here
                 try {
                     byte[] encrypted = EncryptionSystem.encrypt(data);
-                    FileOutputStream fos = new FileOutputStream(Environment.getExternalStorageDirectory() + "/passable." + System.currentTimeMillis());
+                    // determine latest number
+                    int maxPassableFile = 0;
+                    for (File f : Environment.getExternalStorageDirectory().listFiles())
```
Comment (pdmholden) [inline app/src/main/java/ca/deflector/passable_shots/MainActivity.java:50]: I think we can avoid inadvertent bugs in the future by putting the contents of `for` inside `{}`. But if you prefer it this way, I don't have a strong objection. I see no other problems. Go ahead and merge as you see fit.

Comment (DrDub) [inline app/src/main/java/ca/deflector/passable_shots/MainActivity.java:50]: A PR is not a place to discuss project-wide coding standards. Send me an email if you want to discuss them, but no, I don't like to add an extra line, it makes less code to fix in the screen.

If you approve the PR, merge it right away!

Comment (pdmholden) [inline app/src/main/java/ca/deflector/passable_shots/MainActivity.java:50]: I disagree. It was a comment about (a) code readability and (b) the potential for future bugs, which both go beyond coding standards. I think it's quite appropriate for CR.

Comment (DrDub) [inline app/src/main/java/ca/deflector/passable_shots/MainActivity.java:50]: OK, let's try that for the next PR then.

**Pull request merged**